src/assets/style.css

```diff
@@ -495,8 +522,7 @@ hr {
     margin-bottom: var(--whole);
   }
   main {
-    width: 100%;
-    max-width: var(--measure);
+    width: 85vw;
```

Comment:
Need to change this to something that doesn't conflict with the nav bar on small screens